controllers/vcdmachine_controller.go

```diff
@@ -510,6 +513,12 @@ func (r *VCDMachineReconciler) reconcileNormal(ctx context.Context, cluster *clu
 		// 	VCDResourceSet can get bloated with VMs if the cluster contains a large number of worker nodes
 	}
 
+	desiredNetworks := append([]string{vcdCluster.Spec.OvdcNetwork}, vcdMachine.Spec.AdditionalOvdcNetworks...)
+	if err = r.reconcileVMNetworks(vdcManager, vApp, vm, desiredNetworks); err != nil {
```

Comment:
I have a basic question: If we put the networks into the vApp, all VMs get visibility to it. Can we do this in the vcdcluster controller itself? That can in fact be an input parameter to `GetOrCreateVApp`. Then we do not need to do the merge etc